src/commands/ExecCommand.js

```diff
@@ -100,7 +100,7 @@ export default class ExecCommand extends Command {
   }
 
   runCommandInPackage(pkg, callback) {
-    ChildProcessUtilities.spawn(this.command, this.args, this.getOpts(pkg), (err) => {
+    ChildProcessUtilities.spawnStreaming(this.command, this.args, this.getOpts(pkg), pkg.name, (err) => {
```

Comment:
This needs to be behind an option, `--stream`, just like [RunCommand does](https://github.com/lerna/lerna/blob/master/src/commands/RunCommand.js#L18-L23). Otherwise, it's a breaking change, and I'd like to avoid that.